src/RouteRegistrar.php

```diff
@@ -125,14 +125,13 @@ protected function registerRoutes(ReflectionClass $class, ClassRouteAttributes $
                     continue;
                 }
 
-                $httpMethod = $attributeClass->method;
+                $httpMethods = $attributeClass->methods;
 
-                $action = $attributeClass->method === '__invoke'
+                $action = in_array('__invoke', $attributeClass->methods)
                     ? $class->getName()
                     : [$class->getName(), $method->getName()];
```

Comment:
@freekmurze I just read into #25, which fixes this part of the code I also stumbled over. The `__invoke` should probably refer to the controller method instead of the HTTP method, so this should be 
```php
$action = $method->getName() === '__invoke'
    ? $class->getName()
    : [$class->getName(), $method->getName()];
```
I'd fix it if you confirm.

Reply:
Go ahead 👍